File: spring_angular-master/angularapp/src/app/model/pedido.ts
import { ItemPedido } from "./itemPedido";

export class Pedido {

    private listaItemPedido : Array<ItemPedido>
    private totalCostoPedido : number;
    private codigo : string


	constructor($totalCostoPedido: number, $listaItemPedido : Array<ItemPedido>) {
        this.totalCostoPedido = $totalCostoPedido;
        this.listaItemPedido = $listaItemPedido;
	}

    /**
     * Getter $totalCostoPedido
     * @return {number}
     */
	public get $totalCostoPedido(): number {
		return this.totalCostoPedido;
	}

    /**
     * Setter $totalCostoPedido
     * @param {number} value
     */
	public set $totalCostoPedido(value: number) {
		this.totalCostoPedido = value;
    }
    
    public get $listaItemPedido(): Array<ItemPedido> {
        return this.listaItemPedido;
      }
    
      public set $listaItemPedido(value: Array<ItemPedido>) {
        this.listaItemPedido = value;
      }

      /**
     * Getter $totalCostoPedido
     * @return {string}
     */
	public get $codigo(): string {
		return this.codigo;
	}

    /**
     * Setter $totalCostoPedido
     * @param {string} value
     */
	public set $codigo(value: string) {
		this.codigo = value;
    }

      

}

File: spring_angular-master/springapp/src/main/java/co/com/springapp/entity/Pedido.java
package co.com.springapp.entity;

import java.io.Serializable;
import java.util.ArrayList;

/**
 * Clase que representa un pedido.
 * @author fbedoya
 */
public class Pedido implements Serializable {

	/**
	 * 
	 */
	private static final long serialVersionUID = 2228106335093009466L;

	/**
	 * Código del pedido.
	 */
	String codigo;
	
	/**
	 * Total costo del pedido.
	 */
	double totalCostoPedido;
	
	/**
	 * Lista con los items del pedido.
	 */
	private ArrayList<ItemPedido> listaItemPedido;
	
	public Pedido() {
	}

	/**
	 * @return the codigo
	 */
	public String getCodigo() {
		return codigo;
	}

	/**
	 * @param codigo the codigo to set
	 */
	public void setCodigo(String codigo) {
		this.codigo = codigo;
	}

	/**
	 * @return the totalCostoPedido
	 */
	public double getTotalCostoPedido() {
		return totalCostoPedido;
	}

	/**
	 * @param totalCostoPedido the totalCostoPedido to set
	 */
	public void setTotalCostoPedido(double totalCostoPedido) {
		this.totalCostoPedido = totalCostoPedido;
	}

	/**
	 * @return the listaItemPedido
	 */
	public ArrayList<ItemPedido> getListaItemPedido() {
		return listaItemPedido;
	}

	/**
	 * @param listaItemPedido the listaItemPedido to set
	 */
	public void setListaItemPedido(ArrayList<ItemPedido> listaItemPedido) {
		this.listaItemPedido = listaItemPedido;
	}

	/**
	 * @return the serialversionuid
	 */
	public static long getSerialversionuid() {
		return serialVersionUID;
	}

}
